code/game/machinery/computer/gulag_teleporter.dm

```diff
@@ -7,33 +7,40 @@
 	req_access = list(ACCESS_ARMORY)
 	circuit = /obj/item/circuitboard/computer/gulag_teleporter_console
 	var/default_goal = 200
-	var/obj/item/card/id/prisoner/id = null
+	var/obj/item/card/id/prisoner/inserted_id
```

Comment:
give this a better name like `prisoner_id`

Reply:
actually just use the scan var you setup on parent computer. just cast when you need to access prisoner id specific shit.

Reply:
Does this comment still apply? I'd love to figure out how I could get this done more neatly with just two `insert_id` and `eject_id` procs, but the current situation is the only one I could figure out how to get working (id_insert_scan, id_insert_modify, id_insert_prisoner).